File: College_shared_code/Reference/LinearSearch.py
arr = []
ch = 'y'
while(ch == 'y'):

    n = int(input("Enter the size of list : "))

    for i in range((n)):
        a = int(input(f"Enter the {i + 1}th element : "))
        arr.append(a)

    print("Elements in the list are :")
    print(arr)

    search = int(input("Enter the intger to be searched : "))
    flag = 0
    for i in range((n)):
        if(search == arr[i]):
            print(f"Element found at index {i}.")
            flag = 1
            break

    if (flag == 0):
        print("Element not present in the list.")

    ch = input("Want to continue(y/n) : ")

File: College_shared_code/DSA/LinearSearch.c
#include <stdio.h>

int main()
{
    int list[100],size,i,SearchElem;

    printf("Enter the number of elements: ");
    scanf("%d",&size);

    printf("Enter any %d integer elements: \n",size);
    for(i = 0; i < size; i++)
    {
        scanf("%d",&list[i]);
    }

    printf("Enter the element to be searched : ");
    scanf("%d",&SearchElem);
  
    for(i = 0; i < size; i++)
    {
        if(SearchElem == list[i])
        {
            printf("Element found at index %d.\n", i);
            break;
        }
    }

    if(i == size)
    {
        printf("The element to be searched is not found in the list.\n");
    }

  return 0;
}
